packages/reference/src/assets/SortableElements.tsx

```diff
@@ -34,25 +34,32 @@ const SortableLink = SortableElement((props: { children: React.ReactElement }) =
   <div className={styles.item}>{props.children}</div>
 ));
 
-export const SortableLinkList = SortableContainer((props: SortableLinkListProps) => (
-  <div className={props.viewType === 'card' ? styles.gridContainter : styles.container}>
-    {props.items.map((item, index) => (
-      <SortableLink disabled={props.isDisabled} key={`${item.sys.id}-${index}`} index={index}>
-        <FetchingWrappedAssetCard
-          {...props}
-          sdk={props.sdk}
-          key={`${item.sys.id}-${index}`}
-          assetId={item.sys.id}
-          onRemove={() => {
-            props.setValue(
-              props.items.filter((_value, i) => {
-                return i !== index;
-              })
-            );
-          }}
-          cardDragHandle={props.isDisabled ? undefined : <DragHandle />}
-        />
-      </SortableLink>
-    ))}
-  </div>
-));
+export const SortableLinkList = SortableContainer((props: SortableLinkListProps) => {
+  const itemsKey = props.items.map((v) => v.sys.id).join(':');
+
+  return React.useMemo(
+    () => (
+      <div className={props.viewType === 'card' ? styles.gridContainter : styles.container}>
+        {props.items.map((item, index) => (
+          <SortableLink disabled={props.isDisabled} key={`${item.sys.id}-${index}`} index={index}>
+            <FetchingWrappedAssetCard
+              {...props}
+              sdk={props.sdk}
+              key={`${item.sys.id}-${index}`}
+              assetId={item.sys.id}
+              onRemove={() => {
+                props.setValue(
+                  props.items.filter((_value, i) => {
+                    return i !== index;
+                  })
+                );
+              }}
+              cardDragHandle={props.isDisabled ? undefined : <DragHandle />}
+            />
+          </SortableLink>
+        ))}
+      </div>
+    ),
+    [itemsKey, props.isDisabled, props.viewType]
```

Comment:
Probably all props must be listed here, but then there's no point in `useMemo`. I need to track what props make the list re-render because when I inspected them, it seemed like all of them should not be changing after the first render (and user actions).